### PR TITLE
Add C# System.Text.Json

### PR DESCRIPTION
**Pull request by verdie-g**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,22 +35,24 @@ It uses [binary search](https://en.wikipedia.org/wiki/Binary_search_algorithm) t
 On my machine (Ubuntu Linux 4.10.0-35-generic SMP x86_64 with 8Gb RAM, 8.4 MB maximum stack size),
 I found the following results, sorted from least nesting to most nesting:
 
-language        | json library                                                | nesting level | file size     | notes                         |
```
Comment (lovasoa) [inline README.md:38]: you don't need to change every line; can you just add a line for C#?

File: README.md [hunk 2]
```diff
@@ -35,22 +35,24 @@ It uses [binary search](https://en.wikipedia.org/wiki/Binary_search_algorithm) t
...
+java            | [Jackson](https://github.com/FasterXML/jackson-core)                             | 6373          | 13   KB       |
+C++             | [nlohmann::json](https://github.com/nlohmann/json)                               | 13787         | 27.6 KB       | segfault
+Nim             | [json](https://nim-lang.org/docs/json.html)                                      | 100000        | 200 KB        | w/ `-d:release`
+go              | `encoding/json`                                                                  | 2581101       | 5.0 MiB       | goroutine stack exceeds 1000000000-byte limit
+ruby            | [Oj](https://github.com/ohler55/oj)                                              | ∞             | ∞             |
+Haskell         | [Aeson](https://hackage.haskell.org/package/aeson)                               | ∞             | ∞             | available RAM is the only limit
+C#              | [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/api/system.text.json) | ∞             | ∞             | with MaxDepth = int.MaxValue
```
Comment (lovasoa) [inline README.md]: We are trying to compare the default options of each parser. Can you put the results of the parser without any options, and add a note about the possibility to set a `MaxDepth` ?

**Review by lovasoa — CHANGES_REQUESTED** [review]
Could you add it to Travis so that the results are reproductible ?

File: README.md (hunk 1 above)
Comment (lovasoa) [reply README.md:38]: Thank you 👍

**Comment by lovasoa** [comment]
@verdie-g : Have you ever used C# with travis CI ? If so, can you add your C# implementation to our `.travis` ? 
I would have done it myself, but I have zero experience with C# on Travis...

**Comment by verdie-g** [comment]
Ok I've force pushed because I was ruining everything. I can try updating the travis file

**Review by lovasoa — CHANGES_REQUESTED** [review]
Can you add this new `C#` implementation to the travis build ?

**Comment by lovasoa** [comment]
> Ok I've force pushed because I was ruining everything. I can try updating the travis file

Thank you ! 👍

**Comment by lovasoa** [comment]
Great thank you ! It looks like the CI reports 65, and not 64. Should I edit the README to set the result to 65 ?

**Comment by lovasoa** [comment]
I took the freedom to do so

**Comment by verdie-g** [comment]
Is the output 65 the max depth or the depth where an error occurs ? 64 should be the max depth.

**Comment by lovasoa** [comment]
It is the depth of the minimal payload the parser can't parse.

**Comment by lovasoa** [comment]
anyway, thank you for your contribution!

**Pull request merged**